docs/redirects.asciidoc

```diff
@@ -7,6 +7,22 @@
 The following pages have moved or been deleted.
 
 --
+[role="exclude",id="configuring-tls"]
+== Encrypt TLS communications in {kib}
+See {ref}/security-basic-setup-https.html[Set up basic security for the Elastic Stack].
```

Comment:
Nit: what do you think about linking straight to the Kibana section? (same goes for the other two redirects below)

```suggestion
See {ref}/security-basic-setup-https.html#encrypt-kibana-http[Set up basic security for the Elastic Stack].
```

Reply:
+1 to that change! I'll make the same change for the other two redirects as well.